data/scripts/actions/others/crystal_warlord_outfit.lua

```diff
@@ -0,0 +1,43 @@
+local config = {
+	[18520] = {female = 513, male = 512, effect = CONST_ME_GIANTICE}, -- gnomish voucher type CB
+	[18521] = {female = 513, male = 512, addon = 1, effect = CONST_ME_GIANTICE}, -- gnomish voucher type CA1
+	[18522] = {female = 513, male = 512, addon = 2, effect = CONST_ME_GIANTICE} -- gnomish voucher type CA2
+}
+
+local crystalWarlord = Action()
```

Comment:
This needs consistency. We don't want hidden scripts active in the datapack.

either do normal function onUse and register in actions.xml or drop it to data/scripts folder where other revscripts are

Reply:
they are not hidden lol tfs 1.x users should expect scripts to be both in old locations and in revscript location, and I'm trying to be consistent since nearly everything I'm doing is being done in the scripts folder...

as for moving everything to scripts, #2663 

or I just missed your point

Reply:
let's wait for official decision regarding ditching xml file in favour of revscriptsys

personally I'm against it because it's really good to keep track to which action/unique/item ids are tied to which scripts without having to open every script in your datapack to tell that